Cosmetics.py

```diff
@@ -1094,6 +1094,17 @@ def patch_music_changes(rom, settings, log, symbols):
     }
 }
 
+# 7.2.0
+patch_sets[0x1F073FDD] = {
```

Comment:
Sorry, forgot to submit this one with the others.

This conflicts with the patch set for ocarina note shuffle (#1885). Can this be incremented to `0x1F073FDE` to maintain consistency between patch files in test branches?

Reply:
Don't make me hit you